CONSOLE ERROR: line 2: 42
Tests that console.error does not throw exception when executed in console on call frame.

Set timer for test function.
Script execution paused.
Script execution resumed.
setTimeout(testFunction, 0)
1
console.error(42)
42 (program):2(anonymous function) (program):2evaluate (program):426testFunction console-error-on-call-frame.html:10
undefined

